database/migrations/2016_04_01_140748_create_invitations_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateInvitationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('invitations', function (Blueprint $table) {
+            $table->increments('id');
+            $table->boolean('status');
```

Comment:
What does this attribute stand for? 
What does the values assigned to it mean (0 -> means ?, 1-> means ?)
It is different than the ticket status right? (they are even of different data types).